common/src/main/java/dev/latvian/mods/kubejs/gui/KubeJSMenu.java

```diff
@@ -15,10 +16,12 @@
 
 public class KubeJSMenu extends AbstractContainerMenu {
 	public static final DeferredRegister<MenuType<?>> MENU_TYPES = DeferredRegister.create("kubejs", Registries.MENU);
-	public static final Supplier<MenuType<KubeJSMenu>> KUBEJS_MENU = MENU_TYPES.register("menu", () -> MiscPlatformHelper.get().createMenuType());
```

Comment:
Why did you remove `final` here if you don't mind me asking?

Reply:
I was planning on also only filling this variable if serverOnly mode was off, like is done for some other registry KubeJS has, but then realised I didn't need to do that and forgot to revert it.